smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -747,15 +857,11 @@ private getId(childDevice) {
 private poll() {
 	def host = getBridgeIP()
 	def uri = "/api/${state.username}/lights/"
-    try {
-		sendHubCommand(new physicalgraph.device.HubAction("""GET ${uri} HTTP/1.1
+	log.debug "GET: $host$uri"
+	sendHubCommand(new physicalgraph.device.HubAction("""GET ${uri} HTTP/1.1
 HOST: ${host}
 
 """, physicalgraph.device.Protocol.LAN, selectedHue))
-	} catch (all) {
```

Comment:
I followed the code path for sendHubCommand and can't see how an exception could be thrown here. I'm assuming this was in an attempt to be thorough (and the log message is copy/pasted and doesn't apply to this situation), but it looks to be unreachable. I also don't see this being done elsewhere for sendHubCommand, so feel ok removing it.